### DISCUSSION TITLE
Can't figure out how to download Pocket Base

### DISCUSSION THREAD

**Discussion by andycorm** [pocketbase/pocketbase#4649] [Q&A]
Hi! I'd like to experiment with Pocket Base, but I absolutely cannot for the life of me figure out how to download it. Can anyone assist?

I found these two sets of links, which initially looked promising, but nothing happens when I click them. Right clicking and going "Save Link As" also doesn't do anything. Any idea what's going on?

I'm on a 2021 MacBook Pro (M1)

![Screenshot 2024-03-27 at 5 09 39 PM](https://github.com/pocketbase/pocketbase/assets/2769725/63c66f28-4142-4ffb-aeca-6b0bca1f324a)

![Screenshot 2024-03-27 at 5 04 47 PM](https://github.com/pocketbase/pocketbase/assets/2769725/3c9eafa2-feed-4509-9b8c-725adab5e2a6)

**Comment by ramadhanafif** [comment]
I'm not sure what is happening on your mac, but I can download the file.

Try on other browser/machine perhaps?

**Comment by CogalTek** [comment]
have you tried to use curl ?

You can try this 
curl -o link
